package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "acmidm-login-service",
-  "version": "0.9.0",
+  "version": "0.9.1",
   "description": "Microservice to login using ACM/IDM with OpenId",
   "dependencies": {
     "@lblod/mu-auth-sudo": "^0.2.0",
```

Comment:
I'd also bump mu-auth-sudo to 0.3.2 so it listens to the same log parameters as the template.